site/queries/pledge/index.ts

```diff
@@ -0,0 +1,12 @@
+import { urls } from "@klimadao/lib/constants";
+import { IS_LOCAL_DEVELOPMENT } from "lib/constants";
+
+const API_BASE_URL = IS_LOCAL_DEVELOPMENT ? "http://localhost:3000" : urls.home;
+
+export const getPledge = (address: string) =>
```

Comment:
Needs return types, it's probably inferring `any`. See my boilerplate example above